packages/hoppscotch-common/src/helpers/RequestRunner.ts

```diff
@@ -154,6 +154,9 @@ export function runRESTRequest$(
           )
 
           if (E.isRight(runResult)) {
+            // set the response in the tab so that multiple tabs can run request simultaneously
+            tab.value.document.response = res
```

Comment:
We have `updateRESTResponse` method in `Request.vue` component which updates the tab document with the response it received from the network call.

So, when we change the tab updating response into tab document from `RequestRunner` works fine. But, if we don't change the tab the it gets updated twice because the component didn't unmount hence the existing code in that component also updates the response